src/client.ts

```diff
@@ -160,13 +161,19 @@ export class UXDClient {
     const mangoPerpMarketConfig = mango.getPerpMarketConfig(
       depository.collateralMintSymbol
     );
-    const maxRebalancingAmountNative = new BN(
-      maxRebalancingAmount * 10 ** depository.quoteMintDecimals
-    );
+    const maxRebalancingAmountNative = uiToNative(maxRebalancingAmount, depository.quoteMintDecimals);
+    const perpSide = polarity == PnLPolarity.Positive ? 'short' : 'long'; //'sell' : 'buy';
+    const limit_price = (
+      await depository.getLimitPrice(
```

Comment:
Ideally this one too in the `Promise.all` of above but let's keep it simple for now

Reply:
also, camelcase